Added new chart

pyroman1ak opened the pull request:
Added new comparison chart with better labeling than previous chart

walterbender commented:
I don't understand what is going on here: why add delete add? Also, shouldn't we be adding some reference/link to the chart in guide/README.md?

pyroman1ak commented:
@pikurasa Asked me to make a PR and then I made it with the file I had done. Then he edited the labeling on the chart and deleted the previous one and added the one @pikurasa edited. Also, are you talking about the keyboard chart in the guide?

pikurasa commented:
> shouldn't we be adding some reference/link to the chart in guide/README.md?

Sounds like a good idea.

> Also, are you talking about the keyboard chart in the guide?

I think walter is talking about referencing the note chart in the guide.

> I don't understand what is going on here: why add delete add? Also, shouldn't we be adding some reference/link to the chart in guide/README.md?

@pyroman1ak you could have probably just replaced (overwrite) the chart and achieved the same results.
